plugin/src/main/java/com/denizenscript/denizen/scripts/commands/player/TeamCommand.java

```diff
@@ -70,156 +72,90 @@ public TeamCommand() {
-                    && !scriptEntry.hasObject("add")) {
-                scriptEntry.addObject("add", arg.asType(ListTag.class));
-            }
-            else if (arg.matchesPrefix("remove")
-                    && !scriptEntry.hasObject("remove")) {
-                scriptEntry.addObject("remove", arg.asType(ListTag.class));
-            }
-            else if (arg.matchesPrefix("prefix")
-                    && !scriptEntry.hasObject("prefix")) {
-                ElementTag prefixElement = arg.asElement();
-                prefix = prefixElement.asString();
-                scriptEntry.addObject("prefix", prefixElement);
-            }
-            else if (arg.matchesPrefix("suffix")
-                    && !scriptEntry.hasObject("suffix")) {
-                ElementTag suffixElement = arg.asElement();
-                suffix = suffixElement.asString();
-                scriptEntry.addObject("suffix", suffixElement);
-            }
-            else if (arg.matchesPrefix("color")
-                    && arg.matchesEnum(ChatColor.class)
-                    && !scriptEntry.hasObject("color")) {
-                scriptEntry.addObject("color", arg.asElement());
-            }
-            else if (arg.matchesPrefix("option")
-                    && !scriptEntry.hasObject("option")
-                    && (arg.matchesEnum(Team.Option.class)
-                    || arg.matches("friendly_fire", "see_invisible"))) {
-                scriptEntry.addObject("option", arg.asElement());
-            }
-            else if (arg.matchesPrefix("status")
-                    && !scriptEntry.hasObject("status")
-                    && arg.matchesEnum(Team.OptionStatus.class)) {
-                scriptEntry.addObject("status", arg.asElement());
-            }
-            else {
-                arg.reportUnhandled();
-            }
-        }
-        if (name == null || name.length() == 0 || name.length() > 16) {
-            throw new InvalidArgumentsException("Must specify a team name between 1 and 16 characters!");
-        }
-        if (!scriptEntry.hasObject("add") && !scriptEntry.hasObject("remove")
-                && !scriptEntry.hasObject("option") && !scriptEntry.hasObject("color")
-                && !scriptEntry.hasObject("prefix") && !scriptEntry.hasObject("suffix")) {
-            throw new InvalidArgumentsException("Must specify something to do with the team!");
-        }
-        if ((prefix != null && prefix.length() > 64) || (suffix != null && suffix.length() > 64)) {
-            throw new InvalidArgumentsException("Prefixes and suffixes must be 64 characters or less!");
-        }
-        if (scriptEntry.hasObject("option") != scriptEntry.hasObject("status")) {
-            throw new InvalidArgumentsException("Option and Status arguments must go together!");
-        }
-        scriptEntry.defaultObject("id", new ElementTag("main"));
-    }
-
-    public static String translateEntry(String entry, TagContext context) {

```

Comment:
This method should've been updated, not removed - it was useful organization of repeated logic that's now just duplicated below